Cython/Utility/CythonFunction.c

```diff
@@ -1608,51 +1608,67 @@ static int __pyx_FusedFunction_init(PyObject *module) {
 //////////////////// ClassMethod.proto ////////////////////
 
 #include "descrobject.h"
-CYTHON_UNUSED static PyObject* __Pyx_Method_ClassMethod(PyObject *method); /*proto*/
+static CYTHON_INLINE int __Pyx_PyMethodDescr_Check(PyObject *method);  /*proto*/
+static PyTypeObject *__Pyx_PyMethodDescr_Type(PyObject *method);  /*proto*/
```

Comment:
These deserve their own utility code section, so that other parts of Cython can use the functions directly.